lib/web_ui/test/canvaskit/initialization/load_manifest_fonts_manually_test.dart

```diff
@@ -0,0 +1,94 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+import 'dart:js_interop';
+import 'dart:typed_data';
+
+import 'package:test/bootstrap/browser.dart';
+import 'package:test/test.dart';
+import 'package:ui/src/engine.dart';
+import 'package:ui/ui.dart' as ui;
+import 'package:ui/ui_web/src/ui_web.dart' as ui_web;
+import 'package:web_engine_tester/golden_tester.dart';
+
+@JS('window.flutterConfiguration')
+external set _jsConfiguration(JsFlutterConfiguration? configuration);
+
+void main() {
+  internalBootstrapBrowserTest(() => testMain);
+}
+
+void testMain() {
+  test('fonts can be loaded manually', () async {
+    _jsConfiguration = null;
```

Comment:
This is removing any configuration set by the test runner. Please use this instead:

```dart
debugOverrideJsConfiguration(
  <String, Object?>{
    'loadManifestFontsBeforeAppMain': false,
  }.jsify() as JsFlutterConfiguration?
);
```

Then `initializeEngineServices` can be called without any params, IIRC.